changelog.txt

```diff
@@ -32,32 +32,51 @@ Fixes:
     - Dart Mk. II can now mine and build
     - Gold Meltdown now shoots 1 laser instead of 5; DPS changed to 2520
 
-v1.0:
-- Added the Eradicator's Lair and Ground One
-- Added a new boss, the Golden Eradicator
-- Added the Ghost Miner Drone and their factory (suggested by [hb] Kapzduke) 
-- Added Dart Mk. II and their pad
-- Added two new items: Research Point and Shiny Alloy
-- Added a new branch to the tech tree
-- Added a Shiny Alloy Smelter
-- Added the Gold Spectre
-- Added Shiny Walls
-- Added the Shiny Drill, an improved airblast drill (suggested by [hb] Kapzduke) 
-- Fixed turrets' booster multipliers
-- Updated Crushed Gold sprite (artist: [hb] Kapzduke) 
-- Updated Gold Drill Station sprites (artist: kostyan) 
-- Updated Gold Forge sprite (artist: kostyan) 
-- Removed Gold Crusher's script and updated its sprite
+v1.0 "Release":
+New items:
+    - Research Point
+    - Shiny Alloy
+New units:
+    - Ghost Miner Drone (suggested by [hb] Kapzduke)
+    - Golden Eradicator
+New mechs:
+    - Dart Mk. II
+New blocks:
+    - Gost Mined Drone Factory
```

Comment:
Ghost Miner Drone Factory

Reply:
Mined bruh

Reply:
If nothing changes in 24 hours, this PR will be closed.